public/assets/css/style.css

```diff
@@ -0,0 +1,66 @@
+.colored {
```

Comment:
Rename that file to `colors.css`, minify it (and forget) and link to neccesary layouts.

Reply:
Leave this place to me.